contracts/src/v0.8/llo-feeds/v0.4.0/DestinationFeeManager.sol

```diff
@@ -225,7 +225,10 @@ contract DestinationFeeManager is IDestinationFeeManager, ConfirmedOwner, TypeAn
     uint256 numberOfNativeFees;
 
     uint256 feesAndRewardsIndex;
+    bytes32 emptyBytes = bytes32(0);
```

Comment:
FYI i fixed this - the solidity compiler isn't the best, it's more gas efficient to store variables both outside the loop and initialise constants in variables such as bytes(0) if used more than once else the compiler recreates them